src/plugins/controls/public/control_group/component/control_frame_component.tsx

```diff
@@ -40,25 +38,26 @@ interface ControlFrameErrorProps {
 const ControlFrameError = ({ error }: ControlFrameErrorProps) => {
   const [isPopoverOpen, setPopoverOpen] = useState(false);
   const popoverButton = (
-    <EuiText className="errorEmbeddableCompact__button" size="xs">
-      <EuiLink
-        className="eui-textTruncate"
-        color="subdued"
-        onClick={() => setPopoverOpen((open) => !open)}
-      >
-        <EuiIcon type="alert" color="danger" />
-        <FormattedMessage
-          id="controls.frame.error.message"
-          defaultMessage="An error has occurred. Read more"
-        />
-      </EuiLink>
-    </EuiText>
+    <EuiButtonEmpty
+      color="danger"
+      iconSize="m"
+      iconType={'alert'}
+      onClick={() => setPopoverOpen((open) => !open)}
+      className={'errorEmbeddableCompact__button'}
+      textProps={{ className: 'errorEmbeddableCompact__text' }}
+    >
+      <FormattedMessage
+        id="controls.frame.error.message"
+        defaultMessage="An error has occurred. Read more"
+      />
+    </EuiButtonEmpty>
```

Comment:
![ErrorStateBeforeAfter](https://user-images.githubusercontent.com/8698078/215577034-1bcea1be-3942-4bd6-9299-58a4978786f4.png)

Reply:
Is it possible to say something more specific about the error in the message?  Is "Read more" a link and if so, what does it link to?

Reply:
The entire area is actually a button, and clicking on it will open a popover with a very brief description of the error. 

I could change this text to be something like `"An error has occurred. Click here to read more."` to be more specific? :eyes: